CHANGELOG.md

```diff
@@ -1,3 +1,84 @@
+## Release 0.16.0
+
+Highlights:
+
+* New network view to quickly ascertain what networks your containers belong to.
```

Comment:
The ContainerSolutions folks called these things "filters" in a screencast I've just seen. I'd be happy with that.